data_structures/binary_tree/basic_binary_tree.py

```diff
@@ -11,6 +11,23 @@ def __init__(self, data: int) -> None:
         self.left: Node | None = None
         self.right: Node | None = None
 
+def display(tree: Node | None) -> None:  # In Order traversal of the tree
+    """
+    >>> root = Node(1)
+    >>> root.left = Node(0)
+    >>> root.right = Node(2)
+    >>> display(root)
+    0
+    1
+    2
+    >>> display(root.right)
+    2
+    """
+    if tree:
+        display(tree.left)
+        print(tree.data)
+        display(tree.right)
+
 
 def display_using_in_order_traversal(root: Node | None, level: int = 0) -> None:
     """
```

Comment:
Please use the same values as lines 16, 17, 18

Reply:
Line 16 and 34 must be the same.
Line 17 and 35 must be the same.
Line 18 and 36 must be the same.